tests/containers/buildah_docker.pm

```diff
@@ -52,8 +52,9 @@ sub run {
                 test_opensuse_based_image(image => "${prefix_img_name}-working-container", runtime => $buildah, version => $version, beta => $beta);
                 # Due to the steps from the test_opensuse_based_image previously,
                 # the image has been committed as refreshed
-
-                build_and_run_image(runtime => $docker, builder => $buildah, base => $iname);
+                # If "SCC_URL" is set, it means we are in not-released SLE host, which means that we can't use
+                # zypper commands inside containers that are not the same version as the host, so we skip this test.
+                build_and_run_image(runtime => $docker, builder => $buildah, base => $iname) unless get_var('SCC_URL');
```

Comment:
I'm nitpicking: What do you think about moving the check to a library function in `lib/container/common.pm` ? Something like

```
sub is_unreleased_SLE() { return get_var('SCC_URL', '') ne ''; }
```

and then use this function everywhere? Functionally it remains exactly the same, but IMHO this is better understandable, as the magic query is hidden behind a comprehensive function name.

This is a style suggestion only, so feel free to ignore :slightly_smiling_face:

Reply:
I like it :)

Reply:
please take a look again